changelog.html

```diff
@@ -55,6 +55,10 @@
 <!-- Record your changes in the trunk here. -->
 <div id="trunk" style="display:none"><!--=TRUNK-BEGIN=-->
 <ul class=image>
+  <li class=rfe>
+    As <a hreef="http://jenkins-ci.org/content/good-bye-java6">promised</a>, shipping
```

Comment:
Typo. Anyway we generally ask the merger, not the submitter, to add the `changelog.html` entry, given the high chance of conflicts.